stdlib/ballerina-mime/src/main/java/org/ballerinalang/mime/util/MimeUtil.java

```diff
@@ -443,6 +460,44 @@ static boolean isNestedPartsAvailable(BStruct bodyPart) {
                 bodyPart.getNativeData(BODY_PARTS) != null;
     }
 
+    /**
+     * Extract overflow configuration settings from server annotations or get the default settings.
+     *
+     * @param context   Represent ballerina context
+     * @param configAnn Server configs
+     * @return A map with overflow settings
+     */
+    public static Map<String, Object> getOverflowSettings(Context context, Annotation configAnn) {
```

Comment:
Shall we remove Context parameter as it has not been used anywhere within the function?